### DISCUSSION TITLE
学生认证不通过

### DISCUSSION THREAD

**Discussion by u8910** [community/community#51425] [GitHub Education]
为什么我真的是学生却无法通过学生认证？

**Comment by SaigyoujiLuna** [comment]
一样， 已经被驳回7次了

**Comment by WJC-POG** [comment]
学生的可以尝试用学信网的资料来申请，但是我要申请教师资格的就更难受了，课表、合约、工作证全试过了。

**Reply by Jackygod66** [reply]
我用学信网也过不了

**Reply by Jackygod66** [reply]
可以了，要转成英文才行，学信网的验证报告

**Reply by WJC-POG** [reply]
是的，需要英文版

**Reply by u8910** [reply]
他说我的学校必须使用学校的邮箱才可以，我们学校申请邮箱都得走关系，哎

**Reply by WJC-POG** [reply]
学校邮箱为什么要走申请，每个学生都会有的，除非学校没有开通

**Reply by Wyz-0601** [reply]
> 可以了，要转成英文才行，学信网的验证报告

但是没有选择proof type的选项怎么办，一直说我不包含学校名称之类的

**Reply by xyfuestc** [reply]
> 学生的可以尝试用学信网的资料来申请，但是我要申请教师资格的就更难受了，课表、合约、工作证全试过了。

你好，我也是高校老师，遇到一个问题就是无法上传文件，通过我的华为手机，点击take a photo，没有任何反应。请问你有这样的问题吗？

**Comment by 66Xuzming** [comment]
You are unlikely to be verified until you have completed your with your full name as it appears in your academic affiliation document plus a short bio.这个什么意思啊

**Reply by WJC-POG** [reply]
就是你在GitHub的个人信息档案不完善

**Comment by SaigyoujiLuna** [comment]
最好通过电脑，手机端的webview组件有时是有问题的

Yifei Xiao ***@***.***> 于 2024年10月28日周一 15:48写道：

**Reply by xyfuestc** [reply]
我本来就是通过Mac Chrome浏览器，但是发现没有任何反应，才选择采用手机端，但是发现华为手机好像还是没反应 -_-!!!

**Comment by SaigyoujiLuna** [comment]
挂梯子呢

Yifei Xiao ***@***.***> 于 2024年11月13日周三 09:16写道：

**Reply by xyfuestc** [reply]
我估计是我的摄像头坏了。看来得找人修一修。当时以为用不上，就没修。哎.....我用的2015年的mac

**Reply by xyfuestc** [reply]
她报了一个错误：Error: No video devices detected
